plugins/modules/monitoring/spectrum_model_attrs.py

```diff
@@ -0,0 +1,526 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2021, Tyler Gates <tgates81@gmail.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: spectrum_model_attrs
+short_description: Enforce a model's attributes in CA Spectrum.
+description:
+    - This module can be used to enforce a model's attributes in CA Spectrum.
+version_added: 2.1.0
+author:
+    - Tyler Gates (@tgates81)
+notes:
+    - Tested on CA Spectrum version 10.4.2.0.189
+    - Model creation and deletion are not possible with this module. For that use M(spectrum_device) instead.
+requirements:
+    - 'python >= 2.7'
+options:
+   url:
+     description:
+     - URL of OneClick server.
+     type: str
+     required: True
+   url_username:
+     description:
+     - OneClick username.
+     type: str
+     required: True
+     aliases: [username]
+   url_password:
+     description:
+     - OneClick password.
+     type: str
+     required: True
+     aliases: [password]
+   use_proxy:
+     description:
+     - if C(no), it will not use a proxy, even if one is defined in
+       an environment variable on the target hosts.
+     default: yes
+     required: False
+     type: bool
+   name:
+     description:
+     - Model name.
+     type: str
+     required: True
+   type:
+     description:
+     - Model type.
+     type: str
+     required: True
+   validate_certs:
+     description:
+     - Validate SSL certificates.
+     type: bool
+     default: yes
+     required: False
+   attributes:
+     description:
+     - A list of attribute names and values to enforce.
+     - All values and parameters are case sensitive and must be provided as strings only.
+     required: True
+     type: list
+     elements: dict
+     suboptions:
+       name:
+         description:
+         - Attribute name OR hex ID.
+         - Not all names are defined, see C(attr_map) var declaration in __init__() for a complete listing.
+         - Hex IDs are the direct identifiers in Spectrum and will always work.
+         - To lookup hex IDs go to the UI..
+         - Locator -> Devices -> By Model Name -> <enter any model> -> Attributes tab.
+         type: str
+         required: True
+       value:
+         description:
+         - Attribute value. Empty strings should be C("") or C(null).
+         type: str
+         required: True
+'''
+
+EXAMPLES = r'''
+- name: Enforce maintenance mode for modelxzy01 with a note about why
+  spectrum_model_attrs:
+    url: "http://oneclick.url.com"
+    username: "{{ oneclick_username }}"
+    password: "{{ oneclick_password }}"
+    name: "modelxyz01"
+    type: "Host_Device"
+    validate_certs: no
```

Comment:
Examples should avoid insecure behavior.
```suggestion
    validate_certs: true
```